integration-tests/runtime-tests/test/tests/oracle/HAL02-Tests.ts

```diff
@@ -272,8 +272,7 @@ describe("HAL02 [Oracle] Tests", function () {
         expect(result1ReportedPrice.toNumber())
           .to.be.equal(result2ReportedPrice.toNumber())
           .to.be.equal(correctPrice.toNumber());
-        expect(result3ReportedPrice.toNumber())
-          .to.be.equal(maliciousPrice.toNumber());
+        expect(result3ReportedPrice.toNumber()).to.be.equal(maliciousPrice.toNumber());
```

Comment:
Am I missing something? Why are we touching this file to remove the line break here?

Reply:
no idea. just run make style.